ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -101,80 +95,31 @@ export interface AccountCreationState {
   invalidEmail: boolean;
   profile: Profile;
   showAllFieldsRequiredError: boolean;
+  showMostInterestedInKnowingBlurb: boolean;
   usernameCheckInProgress: boolean;
   usernameConflictError: boolean;
-  // TODO(RW-4361): remove all after this point, after we switch to verified institutional affiliation
-  rolesOptions: any;
-  showInstitution: boolean;
-  showNonAcademicAffiliationRole: boolean;
-  showNonAcademicAffiliationOther: boolean;
-  showMostInterestedInKnowingBlurb: boolean;
```

Comment:
This is not related to institutional affiliation and was added after the comment